pkg/render/manager_test.go

```diff
@@ -126,6 +127,11 @@ var _ = Describe("Tigera Secure Manager rendering tests", func() {
 		Expect(d.Spec.Template.Spec.Containers[0].Name).To(Equal("tigera-manager"))
 		Expect(d.Spec.Template.Spec.Containers[0].Env[8].Name).To(Equal("CNX_POLICY_RECOMMENDATION_SUPPORT"))
 		Expect(d.Spec.Template.Spec.Containers[0].Env[8].Value).To(Equal("true"))
+
+		clusterRole := GetResource(resources, render.ManagerClusterRole, "", "rbac.authorization.k8s.io", "v1", "ClusterRole").(*rbacv1.ClusterRole)
+		Expect(len(clusterRole.Rules)).To(Equal(2))
```

Comment:
Please switch this to and the 2 expects below to just one that checks the whole rules has the proper elements.
`Expect(clusterRole.Rules).To(Contain(<slice of expected rules>))`
That will check the 'len' and if there are ever any additions change to the Rules then updating the one Expect will cover the len change and the rule change. Also if we do have an error here, the error will include what was expected and the actual, making it more obvious what the problem was.
(I'm noticing this because of some recent issues with some rule tests).